packages/esm-patient-outcomes-app/src/smsform/send-sms-form.component.tsx

```diff
@@ -7,12 +7,21 @@ import { first } from 'rxjs/operators';
 import { useTranslation } from 'react-i18next';
 import { z } from 'zod';
 import { zodResolver } from '@hookform/resolvers/zod';
-import { ExtensionSlot, showSnackbar, useConnectivity, useLayoutType, usePatient } from '@openmrs/esm-framework';
+import {
+  ExtensionSlot,
+  showSnackbar,
+  useConnectivity,
+  useLayoutType,
+  usePatient,
+  useSession,
+} from '@openmrs/esm-framework';
 import { type DefaultPatientWorkspaceProps } from '@openmrs/esm-patient-common-lib';
 import { type SmsFormData } from './common/types';
 import styles from './send-sms-form.scss';
 import { saveQuestionnaire } from './common';
 import SendSmsField from './send-sms-input.componet';
+import { Select } from '@carbon/react';
+import { SelectItem } from '@carbon/react';
```

Comment:
Combine these two imports. Also it's best to keep the library imports before the local file imports.